### ISSUE TITLE
Sympa - accessibility review and css updates to improve accessibility

### ISSUE THREAD

**Issue by tmclaren** [sympa-community/sympa#1460]
With the rollout of Sympa 6.2 on the University of Illinois campus, we were required to do an accessibility review to determine how the Sympa UI supports individuals with disabilities.    This activity resulted in three results.
1) internal group review and report
2) campus working group review and report
3) we initiated some changes to address issues in style.css.

Expected Behavior
-----------------
Individuals with disabilities should be able to navigate the UI with alternative means, e.g. screen readers, etc.

Current Behavior
----------------
A number of issues where identified where accessibility can be improved.

Possible Solution
-----------------
Please review the attached documents for the accessibility reviews and css changes sans the U of I's color scheme.  However, please notice the comments about color contrasts for any color palette that work best for visually impaired users.

Context
-------
This affects all users with disabilities when navigating the web UI.

**Comment by tmclaren** [comment]
Group Review
[Accessibility Report for Sympa-with comments.docx](https://github.com/sympa-community/sympa/files/9338559/Accessibility.Report.for.Sympa-with.comments.docx)

**Comment by tmclaren** [comment]
Campus Accessibility Group Review
[Sympa 6.2.68 - Campus accessibility report.docx](https://github.com/sympa-community/sympa/files/9338616/Sympa.6.2.68.-.Campus.accessibility.report.docx)

**Comment by tmclaren** [comment]
University of Illinois - Technology Services Group mitigations
[Sympa 6.2.68 - Sympa Accessibility Mitigations.docx](https://github.com/sympa-community/sympa/files/9338637/Sympa.6.2.68.-.Sympa.Accessibility.Mitigations.docx)

**Comment by tmclaren** [comment]
Updated style.css file
[style.css.txt](https://github.com/sympa-community/sympa/files/9338642/style.css.txt)

**Comment by ikedas** [comment]
Hi @tmclaren ,
Thank you and folks of Illinois for recommendations based on detailed survey!

I haven't finished reading all the reviews yet, but for now, about the style sheet suggestions:

  * The stylesheet you based on looks older than 6.2.68: Fxes including #1308, by which web fonts are no longer used in the body text, seem not reflected.

  * Could you please submit your suggestion on stylesheet as a pull request for fixes to the repository?
